apps/agent-provisioning/AFJ/scripts/start_agent_ecs.sh

```
@@ -262,6 +270,67 @@ if [ $? -eq 0 ]; then
   done
+# Describe the ECS service and filter by service name
+service_description=$(aws ecs describe-services --service $SERVICE_NAME --cluster $CLUSTER_NAME --region $AWS_PUBLIC_REGION)
+echo "service_description=$service_description"
+
+
+# Extract Task ID from the service description events
+task_id=$(echo "$service_description" | jq -r '.services[0].events[] | select(.message | test("has started 1 tasks")) | .message | capture("\\(task (?<id>[^)]+)\\)") | .id')
+#echo "task_id=$task_id"
+
+# to fetch log group of container 
+.............................................................
```

Shellcheck (0.10.0)

[warning] This is interpreted as a command name ending with '.'. Double check syntax.

(SC2288)